pom.xml

```diff
@@ -43,11 +44,6 @@
     </plugins>
   </build>
   <dependencies>
-    <dependency>
```

Comment:
removed our Writable impl since we provide a serializer whose output can easily be wrapped in Writable for those who are interested